i18n/tr/cachyos_hello.ftl

```diff
@@ -0,0 +1,82 @@
+# About dialog
+about-dialog-title = CachyOS Hello
+about-dialog-comments = CachyOS Hoşgeldin Ekranı
+
+# Tweaks page
+tweaks = İyileştirmeler
+fixes = Hata düzeltmeler
+applications = Uygulamalar
+removed-db-lock = Pacman veri tabanı kilidi kaldırıldı!
+lock-doesnt-exist = Pacman veri tabanı kilidi bulunmamaktadır!
+orphans-not-found = Hiçbir öksüz paket bulunamadı!
+package-not-installed = '{$package_name}' paketi kurulmamıştır!
+
+# Application Browser page
+advanced-btn = gelişmiş
+reset-btn = sıfırla
+update-system-app-btn = SİSTEMİ GÜNCELLE
+application-column = Uygulama
+description-column = Açıklama
+install-remove-column = Kur/Kaldır
+advanced-btn-tooltip = Genişletilmiş paket seçeneklerini aç veya kapa
+reset-btn-tooltip = Mevcut seçimleri eski haline getir…
+update-system-app-btn-tooltip = Mevcut seçimleri sisteme uygula
+
+# Dns Connections page
+dns-settings = DNS ayarları
+select-connection = Bağlantı seç:
+select-dns-server = DNS sunucusu seç:
+apply = Uygula
+reset = Sıfırla
+dns-server-changed = DNS sunucusu başarıyla değiştirildi!
+dns-server-failed = DNS sunucusu değiştirme işlemi başarısız oldu!
+dns-server-reset = DNS sunucusu eski haline getirildi!
+dns-server-reset-failed = DNS sunucusu sıfırlama işlemi başarısız oldu!
+
+# Tweaks page (tweaks)
+tweak-enabled-title = {$tweak} izin verildi
```

Comment:
"izin verildi" is incorrect, "aktif hale getirildi" is a more correct translation.